windrose/windrose.py

```diff
@@ -20,6 +19,15 @@
 DEFAULT_THETA_LABELS = ["E", "N-E", "N", "N-W", "W", "S-W", "S", "S-E"]
 
 
+def _copy_docstring(source):
```

Comment:
This is the right thing to do b/c mpl made it an internal function. The only thing I'd like to see here is a reference to the original function. I'm not sure if your version is from mpl, stackoverflow, or original. Do you mind adding a docstring with some refs?

Reply:
okay, I added a reference to the `matplotlib` implementation.